cfme/tests/automate/test_file_import.py

```diff
@@ -90,3 +92,63 @@ def test_upload_blank_file(appliance, upload_file):
             view.import_file.upload_file.fill(datastore.file_path)
             view.import_file.upload.click()
             view.flash.assert_message("Error: import processing failed: domain: *")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1753586])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize(
+    "import_data",
+    [
+        DatastoreImport("bz_1753586_user.zip", "bz_1753586_user", None),
+        DatastoreImport("bz_1753586_user_locked.zip", "bz_1753586_user_locked", None),
+        DatastoreImport("bz_1753586_system.zip", "bz_1753586_system", None),
+    ],
+    ids=["user", "user_locked", "system"],
+)
+def test_crud_imported_domains(import_data, temp_appliance_preconfig):
+    """
+    Bugzilla:
+        1753586
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. Export the appliance domain as a zip file
+            2. Extracted the ManageIQ directory to a new dir on the local machine called
+               Domain_Test02
+            3. From within the new Domain_Test02 dir, edit the __domain__.yaml file so that the
+               'name' attribute equals Domain_Test02
+               NOTE: Do not change the 'source' attribute from 'system' to 'user' this time.
+            4. Archived the new Domain_Test02 dir and upload to CloudForms Appliance
+        expectedResults:
+            1.
+            2.
+            3.
+            4. The imported domains should be Custom Domains or these imported domains should gets
+               deleted from db
```

Comment:
The test case is simple enough and I really don't think we need testSteps, besides testSteps doesn't seem to be related to what your test case is actually doing there. I'd rather the testSteps be removed, they are a little confusing. Let me know what you think.

Reply:
Removed testSteps and expectedResults.